build/import/Versions.props

```diff
@@ -3,7 +3,7 @@
   <PropertyGroup>
     <!-- IMPORTANT: These assembly versions must always be 3 digits -->
     <MinimumRequiredVSVersion>16.8.1</MinimumRequiredVSVersion> <!-- We should try to keep this up to date with the highest version available on build machines -->
-    <ProjectSystemVersion>16.10.0</ProjectSystemVersion> <!-- This should always match with the VS release we're targeting -->
+    <ProjectSystemVersion>17.0.1</ProjectSystemVersion> <!-- This should always match with the VS release we're targeting -->
```

Comment:
Should we bump this to 17.0.2 for Preview 2 and beyond?

Reply:
Preview 1 is 17.0.0 so I just bumped it up one but would it make more sense to go to 17.0.2?

Reply:
No, that's fine. Just ensuring that they are unique (and incrementing) is enough. I don't want to have more version numbers than necessary. :-)